src/components/MainPage.tsx

```diff
@@ -96,17 +97,21 @@ function MainPage() {
           setSortByFilter={setSortByFilter}
         />
         <div className='px-4 pb-4 md:px-0 md:pb-0'>
-          {filteredFeedbackList.map((feedback) => (
-            <Link key={feedback.id} to={`/feedback/${feedback.id}`}>
-              <Card
-                title={feedback.title}
-                description={feedback.description}
-                comment={feedback.commentsCount}
-                category={feedback.category}
-                vote={feedback.votesCount}
-              ></Card>
-            </Link>
-          ))}
+          {filteredFeedbackList.length === 0 ? (
+            <NoFeedback />
+          ) : (
+            filteredFeedbackList.map((feedback) => (
+              <Link key={feedback.id} to={`/feedback/${feedback.id}`}>
+                <Card
+                  title={feedback.title}
+                  description={feedback.description}
+                  comment={feedback.commentsCount}
+                  category={feedback.category}
+                  vote={feedback.votesCount}
+                ></Card>
+              </Link>
+            ))
+          )}
```

Comment:
How about this? It seems to be a bit cleaner to me.

```suggestion
          {!filteredFeedbackList.length && <NoFeedback />}
          {filteredFeedbackList?.map((feedback) => (
            <Link key={feedback.id} to={`/feedback/${feedback.id}`}>
              <Card
                title={feedback.title}
                description={feedback.description}
                comment={feedback.commentsCount}
                category={feedback.category}
                vote={feedback.votesCount}
              ></Card>
            </Link>
          ))}
```